tests/expectations/test_expectation.py

```diff
@@ -212,3 +213,32 @@ def test_validate_dependencies_against_available_metrics_failure(metrics_dict):
             metrics=metrics_dict,
             configuration=expectation_configuration,
         )
+
+
+@pytest.mark.unit
+def test_expectation_configuration_stays_up_to_date():
+    expectation = ExpectColumnMaxToBeBetween(column="foo", min_value=0, max_value=10)
+
+    assert expectation.configuration == ExpectationConfiguration(
```

Comment:
very nit, so ignore me if you want. This feels like 2 tests. One ensures the values passed in the constructor are represented in the configuration property, and then the the second half is about staying up to date.